trillian/migrillian/core/trillian.go

```diff
@@ -42,8 +41,8 @@ var errRetry = errors.New("retry")
 // pre-ordered log tree.
 type PreorderedLogClient struct {
 	cli    trillian.TrillianLogClient
-	hasher hashers.LogHasher
-	tree   *trillian.Tree
+	hasher *hasher.Hasher
+	logID  int64
```

Comment:
nit: I'd prefer `treeID` for consistency.

Reply:
It said TreeId where it's set, and LogId where it's used, had to pick the wrong one!

Reply:
Indeed. I guess both options are equally inconsistent :)